src/server/ApiRoutes/ApiRoutes.js

```diff
@@ -11,6 +11,8 @@ import appConfig from '../../../appConfig.js';
 const router = express.Router();
 const appEnvironment = process.env.APP_ENV || 'production';
 
+const apiBase = appConfig.api[appEnvironment];
```

Comment:
This is great, but we should note that we now need to start the app using `APP_ENV=development npm start` or `APP_ENV=production npm start` in the readme.

Also, there's no value in `appConfig.js` for `api['production']` which should be added, even if it's the same url, for now. I think this is why @ablwr was running into problems when running the branch.